src/smart-components/portfolio/portfolio-item.js

```diff
@@ -11,34 +11,49 @@ import {
   StyledGalleryItem
 } from '../../presentational-components/styled-components/styled-gallery';
 import styled from 'styled-components';
+import useFormatMessage from '../../utilities/use-format-message';
+import labelMessages from '../../messages/labels.messages';
 
 const StyledLevel = styled(Level)`
   flex: 1;
 `;
 
-const PortfolioItem = (props) => (
-  <StyledGalleryItem isDisabled={props.removeInProgress && props.isSelected}>
-    <StyledCard>
-      <CardHeader>
-        <StyledLevel>
-          <CardIcon
-            src={`${CATALOG_API_BASE}/portfolio_items/${props.id}/icon`}
-            sourceId={props.service_offering_source_ref}
-          />
-          {props.isSelectable && (
-            <CardCheckbox
-              handleCheck={() => props.onSelect(props.id)}
-              isChecked={props.isSelected}
-              id={props.id}
+const PortfolioItem = (props) => {
+  const formatMessage = useFormatMessage();
+  return (
+    <StyledGalleryItem isDisabled={props.removeInProgress && props.isSelected}>
+      <StyledCard>
+        <CardHeader>
+          <StyledLevel>
+            <CardIcon
+              src={`${CATALOG_API_BASE}/portfolio_items/${props.id}/icon`}
+              sourceId={props.service_offering_source_ref}
             />
+            {props.isSelectable && (
+              <CardCheckbox
+                handleCheck={() => props.onSelect(props.id)}
+                isChecked={props.isSelected}
+                id={props.id}
+              />
+            )}
+          </StyledLevel>
+        </CardHeader>
+        <ServiceOfferingCardBody {...props} />
+        <CardFooter>
+          {props.metadata &&
```

Comment:
You can use conditionals instead: `props?.metadata?.statistics?.approval_processes >= 0 `